src/libnl3/Makefile

```diff
@@ -4,6 +4,7 @@ SHELL = /bin/bash
 
 MAIN_TARGET = libnl-3-200_$(LIBNL3_VERSION)_amd64.deb
 DERIVED_TARGETS = libnl-3-dev_$(LIBNL3_VERSION)_amd64.deb \
+		  libnl-3-200-dbg_$(LIBNL3_VERSION)_amd64.deb \
```

Comment:
why adding dbg here?

Reply:
Have been seeing issues related to netlink, this to get the dbg file copied to target/debs folder. It makes it easier to find the libnl debug package and install it when needed.